backend/attendance/Attendance-Service-2.0.0.yaml

```
@@ -0,0 +1,737 @@
+          items:
+            allOf:
+              - $ref: '#/components/schemas/IndividualEntry'
+              - description: 'Entries of the attendees of the register.'
+        localityCode:
+          type: string
+          example: HIERARCHYTEST_MO_13_02_01_01_23_LIPIRANI
+          description: A reference to the boundary code associated with the attendance register, representing the geographical or administrative area the register is bounded by.
+        reviewStatus:
+          type: string
+          example: PENDINGFORAPPROVAL
+          description:  Indicates whether the attendance register has been reviewed or approved
```

YAMLlint (1.37.1)

[error] too many spaces after colon

(colons)